### PR TITLE
Implement an endpoint that provides information for the block header

### LINKED ISSUE #276: Add an endpoint that provides information for the block header
It provides information about the block header so that the wallet can calculate the "delay block" of pending transactions.

### PR DESCRIPTION
**Pull request by MichaelKim20**
I created this PR to calculate the value for "delay block" for the pending transaction in Agora wallet.

### CODE REVIEW

File: src/modules/storage/LedgerStorage.ts [hunk 1]
```diff
@@ -1445,4 +1445,28 @@ export class LedgerStorage extends Storages
...
         });
     }
+
+    /**
+     * Gets the information of block header
+     * @returns Returns the Promise. If it is finished successfully the `.then`
+     * of the returned Promise is called with the records
+     * and if an error occurs the `.catch` is called with an error.
+     */
+    public getWalletBlocksHeaderInfo (height: Height | null): Promise<any[]>
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts]: A parameter document is required.

File: src/Stoa.ts [hunk 2]
```diff
@@ -828,6 +829,55 @@ class Stoa extends WebService
             );
     }
 
+    /**
+     * GET /wallet/blocks/header
+     *
+     * Called when a request is received through the `/wallet/blocks/header`
+     *
+     * Returns the information abort the block header.
```
Comment (TrustHenry) [inline src/Stoa.ts]: Need a more detailed explanation.

File: src/Stoa.ts [hunk 3]
```diff
@@ -828,6 +829,55 @@ class Stoa extends WebService
...
+                    res.status(204).send(`No blocks`);
+                    return;
+                }
+
+                let info = {
+                    height: rows[0].height.toString(),
+                    hash: new Hash(rows[0].hash, Endian.Little).toString(),
+                    merkle_root: new Hash(rows[0].merkle_root, Endian.Little).toString(),
+                    time_stamp: rows[0].time_stamp
+                };
```
Comment (TrustHenry) [inline src/Stoa.ts:872]: It does not contain all the information in the `block header` like the function name`getWalletBlocksHeader`.

Comment (MichaelKim20) [reply src/Stoa.ts:872]: > It does not contain all the information in the `block header` like the function name`getWalletBlocksHeader`.

I will upgrade after random_seed, missing_validators, and merkle_tree are added to the database.
Currently, I only put what will be used in agora wallet.

File: src/Stoa.ts (hunk 2 above)
Comment (MichaelKim20) [reply src/Stoa.ts]: > Need a more detailed explanation.

I added

File: src/modules/storage/LedgerStorage.ts (hunk 1 above)
Comment (MichaelKim20) [reply src/modules/storage/LedgerStorage.ts]: I added

**Comment by MichaelKim20** [comment]
@TrustHenry Thank for review

**Review by TrustHenry — APPROVED** [review]

**Pull request merged**